pages/1.11/security/ent/perms-reference/index.md

```diff
@@ -99,10 +99,9 @@ Applications launched with Root Marathon cannot receive offers with resources re
 |-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------|---|---|---|---|
 | `dcos:service:marathon:marathon:admin:config`<br>  Controls access to the [GET /v2/info Marathon endpoint](/1.11/deploying-services/marathon-api/#/info).                                                                                                                         |      |   | x |   |   |
 | `dcos:service:marathon:marathon:admin:events` <br>Controls view access to the Marathon events endpoint [GET v2/events](/1.11/deploying-services/marathon-api/#/events).                                                                                                           |      |   | x |   |   |
-| `dcos:service:marathon:marathon:admin:leader` <br> Controls access to the [GET/DELETE /v2/leader](/1.11/deploying-services/marathon-api/#/leader) endpoint.                                                                                                                       | x    |   | x | x |   |
-| `dcos:service:marathon:marathon:services:/[<service-group>]` Controls access to [DC/OS services](/1.11/deploying-services) launched by the native Marathon instance.                                                                                                                | x    | x | x | x | x |
-| `dcos:service:metronome:metronome:jobs[:<job-group>]`<br>  Controls access to [jobs and job groups](/1.11/deploying-jobs/).                                                                                                                                                        | x    | x | x | x | x |
-
+| `dcos:service:marathon:marathon:admin:leader` <br> Controls access to the [GET/DELETE /v2/leader](/1.11/deploying-services/marathon-api/#/leader) endpoint.                                                                                                                       |      |   | x | x |   |
+| `dcos:service:marathon:marathon:services:/[<service-group>]` Controls access to [DC/OS services](/1.11/deploying-services) launched by the native Marathon instance.                                                                                                              | x    | x | x | x | x |
```

Comment:
Please add a `<br>` before "Controls" (missing in original as well).

Reply:
done 😄